README.md

```diff
@@ -1 +1,5 @@
 This is the new repo for the Chassis ThemeRoller tool.
+
+The themeroller will  allow designing and editing themes by customizing existing [css-chassis](http://css-chassis.com) components and is currently under development. To know more, visit this proposal:
```

Comment:
I often refer to it as the proper noun: Chassis, which is how the logo refers to it and the documentation at css-chassis.com.